src/implementations/vanilla/components/menu/_menu.scss

```diff
@@ -521,6 +529,46 @@ $_title-font-weight: null !default;
       background-color: transparent;
     }
 
+    // Previous / Next items
+    .ecl-menu__items-previous {
```

Comment:
I don't really see what we could do to avoid that.
The next/previous buttons are displayed on top of the menu (when needed), so in some cases the last item would be partially hidden below it, sometimes it would go as on your screenshot where most of it is displayed. But that's not really something that we can control